test/fluree/http_api/integration/basic_transaction_test.clj

```diff
@@ -11,11 +11,11 @@
     (let [ledger-name (str "create-endpoint-" (random-uuid))
           address     (str "fluree:memory://" ledger-name "/main/head")
           req         (json/write-value-as-string
-                       {"ledger"         ledger-name
-                        "defaultContext" ["" {"foo" "http://foobar.com/"}]
-                        "txn"            [{"id"      "ex:create-test"
-                                           "type"    "foo:test"
-                                           "ex:name" "create-endpoint-test"}]})
+                       {"@id"      ledger-name
+                        "@context" ["" {"foo" "http://foobar.com/"}]
```

Comment:
I think I may be confused, I expected these `defaultContext`s to change to the [default context iri](https://github.com/fluree/db/blob/main/src/fluree/db/constants.cljc#L36), rather than to `@context`

Reply:
It should work either way in this case, right? I did change one of them but I was thinking it would be good to make sure these got merged into the transaction context too. But if it's better to use the defaultContext IRI here too I'd be happy to.

Reply:
I actually missed that you did change one of them, I see that now. 😄  I had thought that they all became `@context`s, and that seemed potentially Meaningful, so I wanted to double-check my understanding. But no, it doesn't matter for what these tests are actually testing.